lib/supavisor/db_handler.ex

```diff
@@ -225,7 +225,7 @@ defmodule Supavisor.DbHandler do
 
   def handle_event(:internal, :check_buffer, :idle, %{buffer: buff} = data) do
     if buff != [] do
-      Logger.warning("Buffer is not empty, try to send #{IO.iodata_length(buff)} bytes")
+      Logger.info("Buffer is not empty, try to send #{IO.iodata_length(buff)} bytes")
```

Comment:
I think our log level is info on the server.

```suggestion
      Logger.debug("Buffer is not empty, try to send #{IO.iodata_length(buff)} bytes")
```

Until we get the dynamic log level setting?